bindings/flow/tester/DirectoryTester.actor.cpp

```diff
@@ -334,17 +334,17 @@ struct DirectoryRemoveIfExistsFunc : InstructionFunc {
 		if(count.getInt(0) == 0) {
 			logOp(format("remove_if_exists %s", pathToString(directory->getPath()).c_str()));
 
-			bool _ = wait(executeMutation(instruction, [this] () {
+			wait(success(executeMutation(instruction, [this] () {
```

Comment:
Does this compile for you? My understanding was that if the return value is ignored the flow compiler will assume that the corresponding future is of type `Future<Void>`

Reply:
Yup, that's correct. That's why there's now a `success` call, which returns `Future<Void>`

Reply:
Oh I completely missed that part. To me it looked like a change that would fix intellisense but break the build - which would be bad for obvious reasons ;)